local-cli/runAndroid/runAndroid.js

```diff
@@ -90,7 +90,7 @@ function buildAndRun(args) {
       'utf8'
     ).match(/package="(.+?)"/)[1];
 
-  const packageNameWithSuffix = args.appIdSuffix ? packageName + '.' + args.appIdSuffix : packageName;
+  const packageNameWithSuffix = args.appId ? args.appId : args.appIdSuffix ? packageName + '.' + args.appIdSuffix : packageName;
```

Comment:
I am okay with anything which makes the code looks obvious. Even adding parentheses and indentations for original solution is enough.